test/smart-wallet/testGroup1.ts

```diff
@@ -339,7 +339,7 @@ describe("Base Wallet Functionality", function () {
     await expect(
       userSCW
         .connect(accounts[0])
-        .execTransaction(transaction, refundInfo, signature)
+        .execTransaction_S6W(transaction, refundInfo, signature)
```

Comment:
just at one place try to get gas usage improvements. by calling execTransaction, execTransaction_S6W both for two different similar transactions.

Reply:
done
<img width="1168" alt="Screen Shot 2023-03-14 at 13 36 01" src="https://user-images.githubusercontent.com/3930375/224974552-87b8d7b5-511a-4e84-9ea4-e634a4e2a86f.png">